tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyTool.java

```diff
@@ -209,21 +209,12 @@ private void serveRunnningNormallyPage(
 
     private void includeCss(HttpServletRequest request, PrintWriter out) {
         final String contextPath = request.getContextPath();
-        final String cb = getCacheBustingExtension(request);
-        out.write(
-                "<link rel='stylesheet' href='" + contextPath + "/surveytool" + cb + ".css' />\n");
         /*
          * Note: cldrForum.css is loaded through webpack
          */
         // bootstrap.min.css -- cf. bootstrap.min.js elsewhere in this file
         out.write(
                 "<link rel='stylesheet' href='//stackpath.bootstrapcdn.com/bootswatch/3.1.1/spacelab/bootstrap.min.css' />\n");
-        out.write(
-                "<link rel='stylesheet' href='"
-                        + contextPath
-                        + "/css/redesign"
-                        + cb
-                        + ".css' />\n");
```

Comment:
are you sure this isn't still needed by any legacy code, such as jsp?

I searched for "redesign" with a shortcut I like to use:

```
$ cldrgrep redesign
tools/cldr-apps/js/src/esm/cldrEvent.mjs: * (this is mostly derived from old redesign.js and should be divided
tools/cldr-apps/js/src/esm/cldrInfo.mjs:const ITEM_INFO_ID = "itemInfo"; // must match redesign.css
tools/cldr-apps/js/src/esm/cldrInfo.mjs:const ITEM_INFO_CLASS = "sidebyside-scrollable"; // must match redesign.css, cldrGui.mjs, DashboardWidget.vue
tools/cldr-apps/src/main/webapp/css/redesign.css:/* redesign.css contains a lot of the 'bootstrap' type css,
tools/cldr-apps/src/main/webapp/css/redesign.css: The following is here in redesign.css temporarily!
tools/cldr-apps/src/main/webapp/css/redesign.css: Note: redesign.css is loaded by SurveyTool.java AFTER surveytool.css and bootstrap, so has higher priority;
```
That confirms it doesn't occur in any jsp files. It points to this comment in redesign.css itself:

```
/***
 The following is here in redesign.css temporarily!
 Note: redesign.css is loaded by SurveyTool.java AFTER surveytool.css and bootstrap, so has higher priority;
 style.css (for webpack, currently empty) is effectively loaded first, has the lowest priority
***/

body,
html {
	width: 100vw !important;
	height: 100vh !important;
	overflow: hidden !important;
	margin: 0 !important;
	padding: 0 !important;
}
```

I might have written part of that comment myself a long time ago. What I'm wondering is, does the relative priority between these css files (redesign.css, surveytool.css, bootstrap, style.css) matter, and if it does matter, does this change preserve it?

Reply:
BTW I have
```
$ cat ~/bin/cldrgrep
#!/bin/bash
egrep -H "$@" tools/cldr-apps/js/package.json
egrep -H "$@" tools/cldr-apps/js/src/* -d skip
egrep -H "$@" tools/cldr-apps/js/src/esm/*
egrep -H "$@" tools/cldr-apps/js/src/views/*
egrep -H "$@" tools/cldr-apps/src/main/webapp/*.css
egrep -H "$@" tools/cldr-apps/src/main/webapp/css/*.css
egrep -H "$@" tools/cldr-apps/src/main/webapp/*.jsp
egrep -H "$@" tools/cldr-apps/src/main/webapp/WEB-INF/tmpl/*.jsp
egrep -H "$@" tools/cldr-apps/src/main/webapp/WEB-INF/jspf/*.jspf
```

Reply:
The JSPs don't go through the 'serve normally' route… they would be loaded from their own jsp path.

This change makes sure our css files are loaded last and so can have the final override.